### DISCUSSION TITLE
Add custom fields to Customer's address info on order summary page

### DISCUSSION THREAD

**Discussion by johndwells** [craftcms/cms#14894] [Ideas] [answered]
We have a custom "Phone Number" that we require our customers to fill out when placeing an order online. In Commerce's order summary page, where it displays the Customer's address information (shipping & billing), this custom field is not displayed.

The only way to access the phone number is to "Edit" the customer, click on one of the address' gear icons, and click "Edit Address".

This is a lot of hoops to jump through to access vital customer information.

Is there a way to ensure that these important custom fields are included in the read-only address view?

**Comment by lukas-jansen** [comment, marked as answer]
You can overwrite the default address formatter, this way it will show up in the CP and everywhere where you use the `|address` filter
https://craftcms.com/docs/5.x/reference/element-types/addresses.html#customizing-the-formatter

**Reply by johndwells** [reply]
Thanks @lukas-jansen! I had a feeling this might already be possible.
